mini-e2e.groovy

```diff
@@ -102,7 +102,7 @@ node('cico-workspace') {
 			).trim()
 
 			withCredentials([usernamePassword(credentialsId: 'container-registry-auth', usernameVariable: 'CREDS_USER', passwordVariable: 'CREDS_PASSWD')]) {
-				ssh "podman login --username=${CREDS_USER} --password='${CREDS_PASSWD}' ${ci_registry}"
+				ssh "podman login --authfile=~/.docker/config.json --username=${CREDS_USER} --password='${CREDS_PASSWD}' ${ci_registry}"
```

Comment:
authfile contains the username and password for login right? we dont need username and password for login

Reply:
`podman login` stores the details in the authfile, and `podman pull` uses `~/.docker/config.json` if authentication is required.